src/App.js

```diff
@@ -24,6 +26,8 @@ const App = () => (
       <span>Users: </span>
       {users.length}
     </p>
+
+    <PostList posts={posts} commentList={comments} users={users} />
```

Comment:
add user and comments to the posts and pass ONE object to PostList (preparedPosts)